src/app/settings/components/clear-cache/clear-cache.component.html

```diff
@@ -0,0 +1 @@
+Clear cache
```

Comment:
Is this component in anticipation of SOF-1754?

Reply:
Yes